package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "tbtc-dapp",
-    "version": "0.17.6-pre",
+    "version": "0.17.6-rc",
```

Comment:
In other modules we're defining `pre`, shouldn't we do the same here?

Reply:
Right, fixed in 8f40fcc1a12ee0a364d5069d64340c896adfd1ea.